packages/code/src/Code.tsx

```diff
@@ -35,7 +34,7 @@ const lineHeight = 24;
 
 const wrapperStyle: Record<Theme, string> = {
   [Theme.Light]: css`
-    border: 1px solid ${variantColors[Theme.Light][1]};
+    border: 1px solid ${variantColors.light[1]};
```

Comment:
I think using `[Theme.Light]` is technically correct—we should prefer to use use the enum where we can